### PR TITLE
EA Sequences

### PR DESCRIPTION
**Pull request by jpaddison3**
And change some fonts away from Times

The largest LW change here is the part where I noticed several times where LW was using Times as a default font. I thought in all those cases I'd rather the EA Forum use a sans serif stack. I introduce a uiSecondary typography, that LW sets to its serifStack and the EA Forum sets to its sansSerifStack.

I suggest using the sans serif stack for the Alignment Forum. I think it looks better in the context of the alignment forum theming. That's what I've put in the theme file, feel free to ask me to change.

image (2)

### CODE REVIEW

File: packages/lesswrong/components/posts/PostsPage/PostsAuthors.tsx [hunk 1]
```diff
@@ -7,7 +7,8 @@ const styles = createStyles(theme => ({
   root: {
     textAlign: 'left',
     display: 'inline',
-    ...theme.typography.postStyle
+    ...theme.typography.postStyle,
+    fontFamily: theme.typography.uiSecondary.fontFamily,
```
Comment (Discordius) [inline packages/lesswrong/components/posts/PostsPage/PostsAuthors.tsx]: This shouldn't be here. `postStyle` already defined a fontFamily.

File: packages/lesswrong/components/sequences/SequencesPage.jsx [hunk 2]
```diff
@@ -31,7 +31,8 @@ const styles = theme => ({
   title: {
     ...theme.typography.postStyle,
     fontVariant: "small-caps",
-    marginTop: 0
+    marginTop: 0,
+    fontFamily: theme.typography.uiSecondary.fontFamily,
```
Comment (Discordius) [inline packages/lesswrong/components/sequences/SequencesPage.jsx]: Same here, if we want to add this here, we should get rid of the postStyle, which does nothing but define a fontFamily.

File: packages/lesswrong/themes/eaTheme.js [hunk 3]
```diff
@@ -186,6 +189,16 @@ const theme = createLWTheme({
         fontSize: '50%',
       },
     },
+    PostsTopSequencesNav: {
+      root: {
+        marginBottom: -8,
+      },
+      title: {
+        textTransform: 'lowercase',
```
Comment (Discordius) [inline packages/lesswrong/themes/eaTheme.js]: Lowercase sure is a choice here. Surprised by that, but your call.

Comment (jpaddison3) [reply packages/lesswrong/themes/eaTheme.js]: It's then smallcapsed

File: packages/lesswrong/components/sequences/SequencesPage.jsx (hunk 2 above)
Comment (jpaddison3) [reply packages/lesswrong/components/sequences/SequencesPage.jsx]: I think we should, I think it's mildly more clear what's going on, and it allows the EA Forum (and AF?) to change to sans serif in places like this in one place.

File: packages/lesswrong/themes/eaTheme.js (hunk 3 above)
Comment (Discordius) [reply packages/lesswrong/themes/eaTheme.js]: Oh, I see.

Comment (jpaddison3) [reply packages/lesswrong/themes/eaTheme.js]: image

Comment (Discordius) [reply packages/lesswrong/themes/eaTheme.js]: Ah, I see. I think I prefer SmallCaps with variations for actual capital letters, but I can see the choice.

File: packages/lesswrong/components/sequences/SequencesPage.jsx (hunk 2 above)
Comment (Discordius) [reply packages/lesswrong/components/sequences/SequencesPage.jsx]: Hmm, ok.

**Comment by Raemon** [comment]
I'm merging since JP can't merge

**Comment by jpaddison3** [comment]
wait

**Comment by jpaddison3** [comment]
uh

**Comment by jpaddison3** [comment]
https://github.com/LessWrong2/Lesswrong2/pull/2898

**Comment by jpaddison3** [comment]
I think the first two were still reasonable comments, my "I think we should" referred to the block of Oli's conditional "if we want to add this here, we should ...". And I noticed a misalignment in the PostsPageActions.

Sorry, I should have mentioned that I was working on addressing the comments, I just had my lunch break.

**Comment by Raemon** [comment]
Whoops. Sorry about that. (It's a bit tricky with non-LW-team-PRs, my preference is that PRs not be approved without actually being done because otherwise it's unclear what their state is, and there isn't any benefit to approving them earlier. That's more of an argument I'm having with Oli :P) 

Meanwhile, apologies for making it a bit more complicated. I think for now correct thing is to submit a new PR with the changes.

**Comment by jpaddison3** [comment]
Did so, already merged. Development velocity over 9000.

**Pull request merged**